src/app/components/mnoe-product-markups-list/mnoe-product-markups-list.coffee

```diff
@@ -62,13 +64,30 @@
       vm.markups.pageChangedCb(vm.markups.nbItems, 1)
       return search
 
+    customerCategory = ->
+      _.map(vm.markups.list, (app) ->
+        app.customer_name = if app.organization
```

Comment:
I don't see `customer_name` being used?
I assume this has been replaced by `showOrgName()` ?